apps/meteor/app/integrations/server/triggers.ts

```diff
@@ -12,6 +13,6 @@ callbacks.add('afterCreateChannel', callbackHandler('roomCreated'), callbacks.pr
 callbacks.add('afterCreatePrivateGroup', callbackHandler('roomCreated'), callbacks.priority.LOW, 'integrations-roomCreated');
 callbacks.add('afterCreateUser', callbackHandler('userCreated'), callbacks.priority.LOW, 'integrations-userCreated');
 callbacks.add('afterJoinRoom', callbackHandler('roomJoined'), callbacks.priority.LOW, 'integrations-roomJoined');
-callbacks.add('afterLeaveRoom', callbackHandler('roomLeft'), callbacks.priority.LOW, 'integrations-roomLeft');
+afterLeaveRoomCallback.add(callbackHandler('roomLeft'), callbacks.priority.LOW, 'integrations-roomLeft');
 callbacks.add('afterRoomArchived', callbackHandler('roomArchived'), callbacks.priority.LOW, 'integrations-roomArchived');
 callbacks.add('afterFileUpload', callbackHandler('fileUploaded'), callbacks.priority.LOW, 'integrations-fileUploaded');
```

Comment:
This introduces a different style to handle this callbacks. Are we sure we want this changes to propagate? Should we update the other callback methods to reflect this new change?

Reply:
I'm glad you noticed, that's exactly the idea, we want and we're going to use the new way, as we have 1000 callbacks and I don't have a way to do that in a pr.  so its going to be something gradual, btw the use of callbacks is deprecated